libs/core/dtos/dateTimeRange.ts

```diff
@@ -0,0 +1,8 @@
+export interface DateTimeRange {
```

Comment:
Use `Dto` suffix for dto names
```suggestion
export interface DateTimeRangeDto {
```